week0003/hoangdangduy/java/Process.java

```diff
@@ -0,0 +1,84 @@
+import java.util.HashMap;
+
+public class Process {
+	private static final String add = "A";
```

Comment:
The constants should be in uppercase so people can read easily:

`ADD/DEPOSIT/WITHDRAW`